livekit-agents/livekit/agents/ipc/job_main.py

```diff
@@ -35,7 +36,26 @@ async def _start(
 ) -> None:
     close_tx, close_rx = aio.channel()  # used by the JobContext to signal shutdown
 
-    cnt = room.connect(args.url, args.token)
+    auto_subscribe = args.auto_subscribe
+    opts = rtc.RoomOptions()
+    if auto_subscribe != AutoSubscribe.SUBSCRIBE_ALL:
+        opts.auto_subscribe = False
```

Comment:
always false?

Reply:
ah it defaults to True, can you make it explicit?

Reply:
I think it's right.
SUBSCRIBE_NONE != SUBSCRIBE_ALL so we set `opts.auto_subscribe = False`. Then in on_track_published we never subscribe. I will adjust so that the handler if never registered if you'd like